core/kits/assets/js/manager.js

```diff
@@ -37,7 +37,11 @@ export default class extends elementorModules.editor.utils.Module {
 			icon: 'eicon-global-settings',
 			title: elementor.translate( 'site_settings' ),
 			type: 'page',
-			callback: () => $e.route( 'panel/global/menu' ),
+			callback: () => {
+				$e.run( 'panel/global/open', {
+					route: $e.routes.getHistory( 'panel' ).reverse()[ 0 ].route,
```

Comment:
Why route to the history item?

Reply:
@iNewLegend can explain, I did this part with his instruction

Reply:
@udidol, you can choose any route as you wish, maybe @matipojo means that the route to back to, can be static?

Reply:
@udidol the `route` parameter is not relevant to here, please remove it.

Reply:
@matipojo @iNewLegend When I remove this parameter, the "Back to Editor" link does not bring me back to the last place I was in before entering Site Settings. With this parameter, it does redirect me back to where I came from (Editing a widget or the hamburger menu - the two places where you can access global settings).